### DISCUSSION TITLE
Hardware Details of agents accessible via API

### DISCUSSION THREAD

**Discussion by Darryl-One-Degree** [amidaware/tacticalrmm#1155] [Q&A]
Is there a way or is it possible to retrieve the hardware details from agents via API as well as the currently logged in user ?

**Comment by wh1te909** [comment]
yes both of those are returned with the following endpoint: `/agents/<agent_id>/`

example: ```https://api.example.com/agents/jbGQvwaSkvngdHJhZpRSvySHpsYnJEhTcTTSOEiJ/```

you can get all agent_ids from `/agents/versions/`
